added more swahili translation

rosteve opened the pull request:
# Description

Added more translation for Child schedule messages. also fixed some trailing space that was causing sentinel to crash in https://github.com/medic/medic-projects/issues/4500
cc  @abbyad @jillrshah

abbyad commented:
Thanks @rosteve for updating the translations! Are you able to remove the translations which are the same as in the [3.0.x branch](https://github.com/medic/medic-webapp/blob/3.0.x/ddocs/medic/_attachments/translations/messages-sw.properties) and recommit? Once that is done we can merge.

rosteve commented:
Hey @abbyad ,Sure, I have now removed the translations which are the same as in the [3.0.x](https://github.com/medic/medic-webapp/blob/3.0.x/ddocs/medic/_attachments/translations/messages-sw.properties) branch in the above commit.
The first three translation  were existing but I have updated them since these are the ones which had a trailing space and which had caused sentinel to break in https://github.com/medic/medic-projects/issues/4500

abbyad commented:
Hi @rosteve, I have reviewed it again. Here are some notes:

- `report.delivery.delivery_code_label` and `report.delivery.pregnancy_outcome_label` seem to have accidentally become `report.delivery.label_delivery_code` and `report.delivery.label_pregnancy_outcome`
- `targets.*` and `schedule.child_health` are completely missing and are needed. If you aren't translating them yet leave them blank so that we know to come back to them.

Once these are complete we can import into POEditor to make sure they are included in subsequent releases of medic-webapp.

garethbowen commented:
@rosteve Can you finish off the last changes requested by Marc? I'd love to get this merged.

garethbowen commented:
Thanks @rosteve !

@abbyad Can you re-review please?

abbyad commented on config/standard/translations/messages-sw.properties:
```diff
@@ -1,18 +1,74 @@
-messages.schedule.postnatal.week_6_overdue = Je, {{patient_name}} ({{patient_id}}) alihudhuria ziara ya baada ya kujifungua ya wiki  ya 6? Tujulishe kwa kutuma   'M {{patient_id}}'. Shukrani!
-
-report.delivery.delivery_code_label = Kodi ya Kujifungua
+# Do not translate this label
```
Is there a reason to keep this?

```suggestion
```

rosteve commented:
Hey @abbyad , just to be sure, should I add back the translation which I had removed [here](https://github.com/medic/medic-webapp/pull/4765#issuecomment-410427060) and then recommit?

abbyad commented:
> should I add back the translation which I had removed here and then recommit?

No need to add any translations that are in 3.0.x, since those are now included directly in the webapp.

abbyad commented:
The ones that are still blank in this PR are actually in the webapp in English, but not yet translated [there](https://github.com/medic/medic-webapp/blob/345066c88286d200767742d799aa6c1f32a032e2/ddocs/medic/_attachments/translations/messages-sw.properties#L1030-L1050), nor in [POEditor for Swahili](https://poeditor.com/projects/po_edit?order=&filter=&id=33025&id_language=154&search=targets.).  That means that they still need to be translated in the Standard translations AND POEditor. 

Please add those and I will re-review. Thanks!

rosteve commented:
Hey @abbyad , I've now added the missing translations that were blank in this PR in the above commit and in Poeditor too

garethbowen commented:
Closing as this is completely replaced by the new PR: https://github.com/medic/medic/pull/5367
